New docs website / Pagination (#1014)
hashicorp/design-system: website/docs/components/pagination/partials/accessibility/accessibility.md

## Conformance rating

<!-- Update conformance rating badge with correct status and remove the others -->
<Doc::Badge @type="success">Conformant</Doc::Badge>

When used as recommended, there should not be any WCAG conformance issues with this component.

<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>

> Explain how the component is conditionally conformant. For example, any component that offers text truncation is conditionally conformant- it is conformant as long as the text truncation feature is not used.

Reviewer: ```suggestion
Explain how the component is conditionally conformant. For example, any component that offers text truncation is conditionally conformant—it is conformant as long as the text truncation feature is not used.
```

Author: @KristinLBradley I'll disregard these comments as this content comes from the template and I believe it's mostly informational. If it needs to be updated, we should update the template directly.

<Doc::Badge @type="critical">Not conformant</Doc::Badge>

> The `Component Name` component is not WCAG-conformant on its own. (Explain how to use it in a conformant way)

Reviewer: ```suggestion
The `Component Name` component is not WCAG-conformant on its own. (Explain how to use it in a conformant way.)
```

## Best practices

### Keyboard navigation

In most cases, the numbered pagination provides a greater user experience. It allows users to jump between pages and always return to the first page or go to the last page without navigating through the pages manually.

> If your product only has cursor pagination, it won't be able to support the numbered variant. Only applications with offset pagination can implement the numbered pagination variant.

Reviewer: ```suggestion
If your product only has cursor pagination, it won’t be able to support the numbered variant. Only applications with offset pagination can implement the numbered pagination variant.
```

> If your product only has cursor pagination, it won't be able to support the numbered variant. Only applications with offset pagination can implement the numbered pagination variant.

Reviewer: ```suggestion
If your product only has cursor pagination, it won’t be able to support the numbered variant. Only applications with offset pagination can implement the numbered pagination variant.
```

> ![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-01.png =50%x50%)
> 
> Focus and move between pagination controls.

Reviewer 2: Have you considered using the inline code style to indicate the keyboard value? Then you can put the related description under it and then the screenshot (sans keyboard shortcut). It feels like a more natural flow. 

```suggestion
`tab`
Focus and move between pagination controls.
![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-01.png =50%x50%)
```

> ![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-02.png =50%x50%)
> 
> Trigger button to navigate to another page.

Reviewer 2: ```suggestion
`enter`
Trigger button to navigate to another page.
![Keyboard navigation](/assets/components/pagination/pagination-keyboard-navigation-02.png =50%x50%)
```

### Heading 3

> {description of best practice}

Reviewer: I assume the above is placeholder text? Will we be adding the final text soon? If we're not sure, maybe it would be better just to delete the placeholder text so we don't forget about it. (But let me know what you think.)
